desktop/cmp/grid/impl/filter/ColumnHeaderFilterModel.ts

```diff
@@ -67,10 +67,15 @@ export class ColumnHeaderFilterModel extends HoistModel {
 
     @computed
     get isCustomFilter() {
-        const {columnCompoundFilter, columnFilters} = this;
+        const {columnCompoundFilter, columnFilters, fieldType} = this;
         if (columnCompoundFilter) return true;
         if (isEmpty(columnFilters)) return false;
-        return columnFilters.some(it => !['=', '!=', 'includes'].includes(it.op));
+        return columnFilters.some(
+            it =>
+                !['=', '!=', 'includes'].includes(it.op) ||
```

Comment:
this code would be easier to grok if we expanded out to equals checks, and did not try to use the javascript 'includes' function .  We are already dealing with an 'includes' filter and it is confusing!